dtfs-central-api/src/v1/validation/utilisation-report-service/selected-fee-record-validator.ts

```diff
@@ -10,3 +11,24 @@ export const validateSelectedFeeRecordsAllHaveSamePaymentCurrency = (selectedFee
     throw new InvalidPayloadError('Selected fee records must all have the same payment currency');
   }
 };
+
+/**
+ * Validates that all selected fee records with payments have the same fee
+ * record payment group and that the fee record IDs match the fee record IDs on
+ * this existing payment group.
+ * @param feeRecordIds - The fee record IDs
+ */
+export const validateThatAllSelectedFeeRecordsWithPaymentsFormACompletePaymentGroup = async (feeRecordIds: number[]) => {
```

Comment:
```suggestion
export const validateAllSelectedFeeRecordsWithPaymentsFormACompletePaymentGroup = async (feeRecordIds: number[]) => {
```

Reply:
I'm a little far detached from this area of the code -- This name feels very long (which might be needed)
Could something like `validateFeeRecordsAreOnePaymentGroup`?

Reply:
Simplified this a little thanks

Kept the `WithPayments` part as if we have multiple selected fee records at `TO_DO` these aren't part of a payment group so the 'complete payment group' wouldn't apply

Reply:
Great job